assets/scripts/fe/src/app/components/public/login/login.component.ts

```diff
@@ -15,14 +15,16 @@ import { SlackService } from '../../../commons/services/auth/slack.service';
 })
 export class LoginComponent implements OnInit {
   private form : LoginForm;
-
+  private stateName = this.trans.params().next; // get statename from next= parameter
```

Comment:
initialize this properly. what is the reason why do you need to put it in a variable?